client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
+				) &&
```

Comment:
On the other hand, it's a very NUX-focused function, so then again I'm not sure about `current-user`.

Sorry about the comment flood, this was a weird train of thought. 😅